application.go

```diff
@@ -41,8 +41,8 @@ type Application struct {
 	// was drawn.
 	afterDraw func(screen tcell.Screen)
 
-	// If this value is true, the application has entered suspended mode.
-	suspended bool
+	// Used for events loop lock during Suspend()
```

Comment:
Maybe a comment such as `Halts the loop during suspended mode .` is better here.